node/trieIterators/stakeValuesProcessor_test.go

```diff
@@ -0,0 +1,379 @@
+package trieIterators
```

Comment:
following go principles, package names should not use camel case

Reply:
I guess we should start refactoring half of the project then. I know it should have been one word there but I guess those 2 words better explain what the package should do.

Reply:
it could have been `trieiterators` but it's fine